### DISCUSSION TITLE
可疑调用svchost

### DISCUSSION THREAD

**Discussion by yasuo214** [pjy612/SteamManifestCache#21] [Question / Answer]
你好，2.13版本打开提示可疑调用svchost，我看了任务管理器和百度搜索相关服务器都没有找到类似程序名，希望下个版本能优化一下。

**Comment by yasuo214** [comment]
抱歉，是不能解压到任何文件夹里，只能解压到桌面，希望能优化以下。。

**Reply by ZEERDEER** [reply]
请检查自身问题
![图片](https://github.com/pjy612/SteamManifestCache/assets/10573857/065da1cf-2097-4c97-b642-366ad2652b8e)

**Reply by yasuo214** [reply]
抱歉，我就算按照你的目录解压也没办法，只能说是安装的系统问题了，不能解压到任何文件夹里，只能直接解压到桌面，而且还不能创建任何文件夹解压到桌面。

**Reply by pjy612** [reply]
解压是可以随便解压的吧，只是 可能你解压到其他地方后，通过 资源管理器 打开应用有提示罢了。。。
你可以尝试开始运行 cmd 然后 跳转到 你本地放 SAI的目录 在里面 运行 SAI。
如果还是有提示 那就等下一版本吧

**Reply by pjy612** [reply]
当然有必要可以重启下电脑

**Reply by yasuo214** [reply]
总之还是感谢作者的回复，重启没用，cmd解决了。
